CHANGELOG.md

```diff
@@ -21,19 +23,27 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
     - This fixes odd issues which can occur when mounting over an existing
       mount point (we mount `.` as `/opt/openoni`, and previously were mounting
       `./docker/data` as `/opt/openoni/data`, effectively shadowing the actual data directory)
-  - All generated ingest artifacts now live in a named volume
+  - All generated ingest artifacts now live in a named volume, `onidata`, which
+    is mounted into multiple containers as `/var/local/onidata`
 
 ### Migration
 
 - If you use any of our plugins, make sure you look over their repositories and
-  get a version that is built for Python 3
-- Custom themes and plugins you've built will need to be fixed for Python 3 -
-  exact fixes are out of scope for ONI, but you can learn a lot from the
-  [Python porting documentation](https://docs.python.org/3/howto/pyporting.html)
-  as well as the [2to3](https://docs.python.org/2/library/2to3.html) tool's
-  documentation.
+  get a version that is built for Django 2.2.  Django 2.2 only supports Python
+  3, so plugins which work with Django 2.2 will work with Python 3.
+- Custom themes and plugins you've built will need to be fixed for Django 2.2,
+  and therefore Python 3
+  - Exact fixes are out of scope for ONI, but you can learn a lot from:
+    - The [Python porting documentation](https://docs.python.org/3/howto/pyporting.html)
+    - The [2to3](https://docs.python.org/2/library/2to3.html) tool's documentation
+    - The [Django upgrade documentation](https://docs.djangoproject.com/en/2.2/howto/upgrade-version/)
+  - Most themes and plugins will need minimal work to get updated, but complex
+    Python code could require a lot of migration effort
```

Comment:
These seems like an entirely subjective statement that doesn't really help someone migrating :thinking:. They'll find out whether it's hard or easy on their own, I'm thinking.

Reply:
It's more of a general warning - if you have a complex plugin, be prepared....